src/time.jl

```diff
@@ -0,0 +1,209 @@
+"""
+    time_units(img)
+
+Returns the units (i.e. Unitful.unit) the time axis is measured in. If not available
+`nothing` is returned.
+"""
+time_units(x::Any) = unit(eltype(timeaxis(x)))
+
+"""
+    echo_time(x) -> Float64
+
+Return the echo time (TE) for the acquisition.
+
+This parameter is REQUIRED if corresponding fieldmap data is present or the
+data comes from a multi echo sequence.
+
+please note that the DICOM term is in milliseconds not seconds
```

Comment:
I agree that the `Unitful` route is probably the best way to go with these. I would like the actual units to be consistent though. Do you think it would make sense for there to be something like `x -> uconvert(u"s", x)` for the default here?

Reply:
I actually think it's fine for the values to be returned in a unit which is any multiple of seconds, provided that's reflected in the type. I think this accurately reflects the way that DSP hardware works, which is to count in *integer* multiples of some internal clock time step.

To the user there's not really much of a cost vs always returning in seconds. They can do `ustrip(u"s", x)` or `uconvert(u"s", x)`, and do arithmetic between compatible units as desired.